src/DynamoCore/Configuration/PathManager.cs

```diff
@@ -196,11 +185,6 @@ public IEnumerable<string> DefinitionDirectories
             get { return RootDirectories.Select(path => TransformPath(path, DefinitionsDirectoryName)); }
         }
 
-        public string CommonDefinitions
```

Comment:
not sure if we can remove public properties ASAP, probably need to obsolete it and make it return empty string first? @mjkkirschner any suggestions?